style.css

```diff
@@ -0,0 +1,30 @@
+body {
+
+margin: 0px;
+
+/*styling*/
+font-family: arial;
+color: red;
+
+}
+
+	#header {
```

Comment:
@samuraiseoul: Is tabbing of sub tags on the stylesheet going against convention? I've been doing it because it helps me keep track of the parent/child relationships of tags, but I can start doing away with it if its frowned upon.